app/assets/javascripts/administrate/components/table.js

```diff
@@ -6,12 +6,17 @@ $(function() {
         event.keyCode == keycodes.space ||
         event.keyCode == keycodes.enter) {
 
-      if(!event.target.href) {
+      if(!isLinkableElement(event.target)) {
         window.location = $(event.target).closest("tr").data("url");
       }
     }
   };
 
+  var isLinkableElement = function(element) {
+    return (event.target.href ||
+            $(element).closest(".cell-data--linkable").length > 0);
```

Comment:
Hey there! I'm just a little unclear on how this behavior works.. Would you mind explaining it to me?